x/stakeibc/keeper/msg_server_rebalance_validators.go

```diff
@@ -35,6 +37,32 @@ func floatmax(a, b float64) float64 {
 	return b
 }
 
+func ValAddressFromBech32CusomPrefix(address string, bech32PrefixValAddr string) (addr sdk.ValAddress, err error) {
+	if len(strings.TrimSpace(address)) == 0 {
+		return sdk.ValAddress{}, errors.New("empty address string is not allowed")
+	}
+
+	bz, err := sdk.GetFromBech32(address, bech32PrefixValAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	return sdk.ValAddress(bz), nil
+}
+
+func AccAddressFromBech32CustomPrefix(address string, bech32PrefixAccAddr string) (addr sdk.AccAddress, err error) {
+	if len(strings.TrimSpace(address)) == 0 {
+		return sdk.AccAddress{}, errors.New("empty address string is not allowed")
+	}
+
+	bz, err := sdk.GetFromBech32(address, bech32PrefixAccAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	return sdk.AccAddress(bz), nil
+}
+
```

Comment:
It doesn't look like these functions are used anymore? If so, let's remove them. Otherwise, might make sense to move them to utils